internal/cli/terraform.go

```diff
@@ -172,7 +173,7 @@ func generateTerraformImportConfig(outputDIR string, data importDataList) error
 		return err
 	}
 
-	return createImportFile(outputDIR, data)
+	return createImportFile(outputDIR, deduplicateResourceNames(data))
```

Comment:
Should this be called here instead? https://github.com/auth0/auth0-cli/blob/3bd51780c7130ced6f508e1775ced9d2b2ad313c/internal/cli/terraform.go#L147

```
func fetchImportData(ctx context.Context, fetchers ...resourceDataFetcher) (importDataList, error) {
	var importData importDataList

	for _, fetcher := range fetchers {
		data, err := fetcher.FetchData(ctx)
		if err != nil {
			return nil, err
		}

		data := deduplicateResourcesNames(data)

		importData = append(importData, data...)
	}

	return importData, nil
}
```

Reply:
Could definitely make sense in a few locations. I'm ok with moving into that function.

Reply:
Actually I decided to move out of the for loop but still within the `fetchImportdata` function. Doing so allowed me to apply a slightly stronger test case.